### PR TITLE
add landing page

### LINKED ISSUE #65: Landing page
While we're working on the homepage, can we have a "coming soon" page just with sign up?

we are now live at djangonaut.space --  we could open the sign up for registration.

Pros:
- It will help us figure out resources if we get people in timezones we don't have covered
- We don't waste the opportunity to get people connected while we're still developing

Cons:
- random sign  ups
- too big of a first session

### PR DESCRIPTION
**Pull request by sabderemane**
https://user-images.githubusercontent.com/17890338/225504152-ae061d79-0163-484f-94ea-aae5b7d68e1c.mov

Notes:
- the particles on the background are animated, render better in live, but I can remove it if it's annoying or something else
- the form is doing nothing right now

closes #65 

EDIT: last version
Capture d’écran 2023-06-27 à 00 16 27

### CODE REVIEW

**Comment by dawnwages** [comment]
Hey Sarah, sorry this has taken so long to review. If I gave you a logo that has transparent background and white text would you prefer updating that to the landing page?

**Comment by dawnwages** [comment]
Hopefully this image works? if there's some image quality issues, I'm happy to try to figure out a solution.

djangonaut_space_logo-2-removebg-preview

**Comment by dawnwages** [comment]
image-removebg-preview

**Comment by sabderemane** [comment]
> Hey Sarah, sorry this has taken so long to review. If I gave you a logo that has transparent background and light text would you prefer updating that to the landing page?
> 

No worries Dawn, Yes it would be better without the background, I would like to try with the red one (as it's the initial one) without the background or grey

File: home/templates/home/home_page.html [hunk 1]
```diff
@@ -7,23 +7,21 @@
...
-{% endwith %}
-{% include 'home/includes/events_past.html' %}
-{% with content_text="We do three session per year. This is completely free and sponsored by Django Software Foundation. You're welcome to join any session at any time but you get the full benefit of the program by starting the session with your peers." %}
-    {% include 'home/includes/homepage_blade.html' with title="How it works" content=content_text color="#cccccc"%}
-{% endwith %}
-
-{% with content_text="<a href='/'> Sign Up Today</a>" %}
-    {% include 'home/includes/homepage_blade.html' with title="What are you waiting for?" content=content_text color="#FFF"%}
-{% endwith %}
+    <h1 class="p-1">Ready to start to contribute?</h1>
```
Comment (sarahboyce) [inline home/templates/home/home_page.html:16]:
```suggestion
    <h1 class="p-1">Ready to launch?</h1>
```

**Review by sarahboyce — APPROVED** [review]
:partying_face: let's get this in

**Pull request merged**